src/components/Person/Person.jsx

```diff
@@ -1 +1,15 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    {person.age && <p className="Person__age">{`I am ${person.age}`}</p>}
+    {person.isMarried ? (
+      <p className="Person__partner">
+        {`${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}`}
```

Comment:
Avoid nesting ternary operators. Let's create a variable for wife/husband
![image](https://github.com/mate-academy/react_person/assets/96409563/1103fb19-cc85-400c-a96e-c16c3261cb62)

Reply:
created isMale boolean variable